src/features/soft_navigations/aggregate/bel-node.js

```diff
@@ -1,12 +1,12 @@
-import { now } from '../../../common/timing/now'
+import { TimeKeeper } from '../../../common/timing/time-keeper'
 
 let nodesSeen = 0
 
 export class BelNode {
   belType
   /** List of other BelNode derivatives. Each children should be of a subclass that implements its own 'serialize' function. */
   children = []
-  start = now()
+  start = TimeKeeper.now()
```

Comment:
This is actually always overwritten by either the `AjaxNode` constructor or the `Interaction` constructor. I think it'd be better to just remove the default
```suggestion
  start
```

Reply:
Can we look at that work as part of another ticket? Do you want to open a ticket for that?